server/http.js

```diff
@@ -33,10 +35,40 @@ const zipper = (tree, req, res, next) => {
 	next()
 }
 
-exports.createServer = (tree) =>
-	http.createServer((req, res) =>
+// Handle '/download/{path}' URLs
+const prefixedDownload = (prefix, root) => {
+	const download = serveStatic(root, {
+		setHeaders: function (res, filename) {
+			// Force download
+			res.setHeader('Content-Description', 'File Transfer')
+			res.setHeader('Content-Type', 'application/octet-stream')
+			res.setHeader('Content-Disposition', 'attachment; filename=' + path.basename(filename))
+			res.setHeader('Content-Transfer-Encoding', 'binary')
+			res.setHeader('Expires', '0')
+			res.setHeader('Cache-Control', 'must-revalidate, post-check=0, pre-check=0')
+			res.setHeader('Pragma', 'public')
+		}
+	})
+
+	const fullPrefix = '/' + prefix.replace(/^\/?(.*?)\/?$/, '$1') + '/'
```

Comment:
The download from the server works great when I try to download `package.json` but fails for `.eslintrc.json`. I suspect there's something going on with hidden files. Maybe this regex?